Tools/AP_Bootloader/board_types.txt

```diff
@@ -267,6 +267,8 @@ AP_HW_ORAQF405PRO                    1155
 AP_HW_CBU_StampH743                  1156
 AP_HW_FOXEERF405_V2                  1157
 
+AP_HW_JHEMCUF405WING                 1160
```

Comment:
I suggest creating a PR which reserves an ID ahead of this PR being merged.  I think we're probably going to merge a board using this ID before this PR is merged, so you'll need a new ID...